### DISCUSSION TITLE
EKG/Principles

### DISCUSSION THREAD

**Discussion by giscus** [EKGF/ekg-principles#19] [Page Comments]
# EKG/Principles

Manifesto and principles for the Enterprise Knowledge Graph (EKG)

https://principles.ekgf.org/

**Comment by jgeluk** [comment]
One proposal I heard is to call this page "Values". Many organizations have published their values like "diversity", "sustainable" etc. Very similar to the these "ideals" here. We don't want to preach but be able to make the link between for example "transparancy" and how EKG capabilities and use cases contribute to that (or not).
